firmware/uf2.ksy

```diff
@@ -0,0 +1,169 @@
+meta:
+  id: uf2
+  title: UF2
+  file-extension: uf2
+  license: CC0-1.0
+  endian: le
+doc: |
+  UF2 is a file format, developed by Microsoft for PXT (also known as
+  Microsoft MakeCode), that is particularly suitable for flashing
+  microcontrollers over MSC (Mass Storage Class; aka removable flash drive).
+
+  The list of family-ids is stored in a separate JSON file:
+
+  <https://github.com/microsoft/uf2/blob/master/utils/uf2families.json>
+
+  This file is regularly updated and the family_id enum should be kept in
```

Comment:
Probably enums should go into a separate file that is generated by a script.

Reply:
Maybe, but it seems that that list gets very few updates, so perhaps we could leave it as it is.